hypothesis-python/docs/packaging.rst

```diff
@@ -30,14 +30,10 @@ Dependencies
 Python versions
 ~~~~~~~~~~~~~~~
 
-Hypothesis is designed to work with a range of Python versions. Currently supported are:
-
-* pypy-2.6.1 (earlier versions of pypy *may* work)
-* CPython 2.7.x
-* CPython 3.4.x
-* CPython 3.5.x
-* CPython 3.6.x
-* CPython 3.7.x
+Hypothesis is designed to work with a range of Python versions.  We always support
+`all versisions of CPython with upstream support <https://devguide.python.org/#status-of-python-branches>`_,
+and plan to drop Python 2 at EOL in 2020.  We also support the latest versions of PyPy
+for Python 2 and Python 3.
```

Comment:
Inconsistent with what it says elsewhere (and our informal plans) that we're dropping Python 2 support in PyPy when we drop it for CPython even though PyPy will continue to support it.